### PR TITLE
Fix CHANGELOG

### PR DESCRIPTION
**Pull request by ericmartinezr**
it got a little broken, so I tried to fix it. 

`<template>` broke part of the changelog.
Removed changelog that was duplcated and inserted into alpha35 changelog (don't know what happened here)
Added breaking changes for constants names in Router

### CODE REVIEW

**Comment by PascalPrecht** [comment]
@ericmartinezr can you squash your commits and rename commit message to sth like `chore(readme): ....` ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 * **http/http:** allow for commonjs as ngHttp ([16eb8ce](https://github.com/angular/angular/commit/16eb8ce)), closes [#3633](https://github.com/angular/angular/issues/3633)
 * **injector:** support getRootInjectors on dehydrated injectors. ([92da543](https://github.com/angular/angular/commit/92da543)), closes [#3760](https://github.com/angular/angular/issues/3760)
 * **injectors:** reset the construction counter in dynamic strategy. ([272ad61](https://github.com/angular/angular/commit/272ad61)), closes [#3635](https://github.com/angular/angular/issues/3635)
-* <template> tag for browsers that do not suppor them ([ddcfd46](https://github.com/angular/angular/commit/ddcfd46)), closes [#3636](https://github.com/angular/angular/issues/3636)
+* **`<template>`** tag for browsers that do not suppor them ([ddcfd46](https://github.com/angular/angular/commit/ddcfd46)), closes [#3636](https://github.com/angular/angular/issues/3636)
```
Comment (zoechi) [inline CHANGELOG.md]: suppor**t** misses a **t**

**Comment by PascalPrecht** [comment]
If you want, I can take your commits, squash them and send a new PR. You'll still be the author of commits

**Comment by ericmartinezr** [comment]
@PascalPrecht Go for it, if I keep trying I'll keep messing it up. Thanks for your help and sorry for so many weird things in the PR :P

**Comment by PascalPrecht** [comment]
You can close this one.

Here's the new PR: https://github.com/angular/angular/pull/3937

**Comment by ericmartinezr** [comment]
Thanks @PascalPrecht ;)

**Pull request closed without merging**